src/Conan.cmake

```diff
@@ -19,10 +19,10 @@ macro(run_conan)
   include(${CMAKE_BINARY_DIR}/conan.cmake)
 
   # Add (or remove) remotes as needed
-  # conan_add_remote(NAME conan-center URL https://conan.bintray.com)
+  # conan_add_remote(NAME conan-center URL https://center.conan.io)
   conan_add_remote(
     NAME
-    cci
+    conancenter
```

Comment:
It uses a different name that the conan default, so basically it renames `conancenter` to `cci` for 99% of users.

This is the conan default: 

https://github.com/conan-io/conan/blob/c3fc425871149b12a1d12153bed90a06a3d20132/conans/client/cache/remote_registry.py#L14

```python
CONAN_CENTER_REMOTE_NAME = "conancenter"
```